### PR TITLE
release 0.0.9

### PR DESCRIPTION
**Pull request by turadg**
- **chore: release 0.0.9**
- **docs: how to publish**

### CODE REVIEW

File: MAINTENANCE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Publishing to NPM
+
+So far there is little process to publish a new version of `@agoric/synthetic-chain` to NPM. (https://github.com/Agoric/agoric-3-proposals/issues/60 is to develop automation.)
+
+1. Bump the version number in `synthetic-chain/package.json`.
+2. PR to get it approved.
+3. When approved, run `npm publish` in the branch and merge.
```
Comment (mhofman) [inline MAINTENANCE.md:7]: Shouldn't we also push a tag then? I don't remember if `npm publish` generates it.

**Review by mhofman — APPROVED** [review]

File: MAINTENANCE.md (hunk 1 above)
Comment (turadg) [reply MAINTENANCE.md:7]: It's not generating tags. We had something making [tags for 0.0.{1,2,3}](https://github.com/Agoric/agoric-3-proposals/tags) but I don't recall what.

Comment (gibson042) [reply MAINTENANCE.md:7]: And also a git tag, note that https://www.npmjs.com/package/@agoric/synthetic-chain?activeTab=versions includes versions that have no representation in https://github.com/Agoric/agoric-3-proposals/tags .

File: MAINTENANCE.md
Comment (gibson042) [inline MAINTENANCE.md:1]: Suggestion: Rename to MAINTAINERS.md for alignment with [agoric-sdk](https://github.com/Agoric/agoric-sdk/blob/master/MAINTAINERS.md).

Comment (turadg) [reply MAINTENANCE.md:1]: ah, what I had in mind. though when I google that I find [most MAINTAINERS.md](https://www.google.com/search?q=maintainers.md&sca_esv=03235397ee0f049b&sca_upv=1&ei=2WQdZtfeN83H0PEP-ra-sAE&ved=0ahUKEwiXr_ig4MSFAxXNIzQIHXqbDxYQ4dUDCBE&uact=5&oq=maintainers.md&gs_lp=Egxnd3Mtd2l6LXNlcnAiDm1haW50YWluZXJzLm1kMgUQABiABDIGEAAYBxgeMgUQABiABDIGEAAYHhgKSKUPUL0DWJcOcAF4AJABAJgBhQGgAekBqgEDMC4yuAEDyAEA-AEBmAICoALwAcICBxAAGIAEGA3CAgYQABgeGA3CAggQABgeGA0YCsICCBAAGAgYHhgNwgIKEAAYCBgeGA0YD5gDAIgGAZIHAzAuMqAHpgc&sclient=gws-wiz-serp) list the maintainers. [RELEASING.md](https://www.google.com/search?q=releasing.md&sca_esv=03235397ee0f049b&sca_upv=1&ei=-GQdZs_QG-nk0PEPjNGa-As&ved=0ahUKEwjPrMCv4MSFAxVpMjQIHYyoBr8Q4dUDCBE&uact=5&oq=releasing.md&gs_lp=Egxnd3Mtd2l6LXNlcnAiDHJlbGVhc2luZy5tZDIGEAAYHhgNMggQABgIGB4YCjIKEAAYCBgeGA0YDzIKEAAYCBgeGA0YD0jGJVC5C1jcJHAJeACQAQCYAWugAdoMqgEEMTYuM7gBA8gBAPgBAZgCHKACng3CAggQABiABBiwA8ICCRAAGAcYHhiwA8ICCRAAGB4YsAMYCsICCxAAGIAEGIoFGJECwgIIEAAYgAQYsQPCAgsQLhiABBixAxiDAcICERAuGIAEGLEDGIMBGMcBGNEDwgILEC4YgAQYxwEY0QPCAgUQLhiABMICEBAAGIAEGIoFGJECGEYY-QHCAg0QABiABBiKBRhDGLEDwgIKEAAYgAQYigUYQ8ICDhAuGIAEGIoFGLEDGIMBwgIqEAAYgAQYigUYkQIYRhj5ARiXBRiMBRjdBBhGGPkBGPQDGPUDGPYD2AEBwgIOEAAYgAQYigUYkQIYyQPCAgsQABiABBiKBRiSA8ICExAAGIAEGIoFGJECGLEDGEYY-QHCAgUQABiABMICEBAuGIAEGIoFGEMYxwEYrwHCAi0QABiABBiKBRiRAhixAxhGGPkBGJcFGIwFGN0EGEYY-QEY9AMY9QMY9gPYAQHCAgsQABiABBixAxiDAcICCBAuGIAEGLEDwgIQEAAYgAQYigUYQxixAxiDAcICCxAAGIAEGIoFGLEDwgILEC4YgAQYxwEYrwHCAgoQABiABBgKGLEDwgINEAAYgAQYChixAxiDAcICBxAAGIAEGArCAhoQLhiABBjHARivARiXBRjcBBjeBBjgBNgBAsICBxAAGIAEGA3CAg0QABiABBgNGLEDGLEDwgIKEAAYgAQYDRixA8ICDRAuGIAEGA0YsQMYsQPCAg0QABiABBiKBRgNGLEDwgIUEAAYgAQYDRixAxixAxgKGEYY-QHCAg0QABiABBgNGLEDGIMBwgIrEAAYgAQYDRixAxgKGEYY-QEYlwUYjAUY3QQYRhj5ARj0Axj1Axj2A9gBAcICCBAAGB4YDRgKwgIPEAAYgAQYsQMYChhGGPkBwgIpEAAYgAQYsQMYChhGGPkBGJcFGIwFGN0EGEYY-QEY9AMY9QMY9gPYAQHCAgwQABiABBgKGEYY-QHCAiYQABiABBgKGEYY-QEYlwUYjAUY3QQYRhj5ARj0Axj1Axj2A9gBAcICBBAAGB7CAgYQABgeGAqYAwCIBgGQBgS6BgYIARABGBO6BgYIAhABGBSSBwQyNC40oAfozAE&sclient=gws-wiz-serp) is what this file is for atm.

File: packages/synthetic-chain/package.json [hunk 2]
```diff
@@ -1,8 +1,10 @@
 {
   "name": "@agoric/synthetic-chain",
-  "version": "0.0.8",
+  "version": "0.0.9",
   "description": "Utilities to build a chain and test proposals atop it",
-  "bin": "dist/cli/cli.js",
+  "bin": {
+    "synthetic-chain": "dist/cli/cli.js"
+  },
```
Comment (mhofman) [inline packages/synthetic-chain/package.json:7]: wasn't there some issue with this we reverted this change before?

Comment (turadg) [reply packages/synthetic-chain/package.json:7]: I don't know whether it was intentional. Do you recall something? Regardless, the `npm` command changes it while publishing so I think this is actually what's happening

**Pull request merged**